project/AppDependencies.scala

```diff
@@ -7,6 +7,7 @@ object AppDependencies extends {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
+    "uk.gov.hmrc"             %% "simple-reactivemongo"     % "7.26.0-play-26",
```

Comment:
Don't forget the dependency updates.

Reply:
will change this to "org.reactivemongo" %% "play2-reactivemongo" % "0.18.6-play26"

Reply:
I meant more updating all the dependencies, play-ui, auth etc.